project/SparkBuild.scala

```diff
@@ -397,6 +397,11 @@ object TestSettings {
     testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a"),
     // Enable Junit testing.
     libraryDependencies += "com.novocode" % "junit-interface" % "0.9" % "test",
+    // Needed for SQL JDBC tests.
+    libraryDependencies += "com.h2database" % "h2" % "1.4.183" % "test",
```

Comment:
build dependencies are now defined in pom files i think. sbt just loads them from the pom file.